common/test/acceptance/pages/studio/overview.py

```diff
@@ -236,12 +236,17 @@ def subsection_expanded():
         currently_expanded = subsection_expanded()
 
         self.q(css=self._bounded_selector('.ui-toggle-expansion i')).first.click()
+        self.wait_for_element_presence(self._bounded_selector(self.ADD_BUTTON_SELECTOR), 'Subsection is expanded')
+
+        self.browser.execute_script("jQuery.fx.off = true;")
```

Comment:
no, it's not needed, I copy pasted this from line above, but forgot to remove it afterwards.